zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -192,9 +217,54 @@ class Thermostat_TYST11_c88teujp(TuyaThermostat):
     }
 
 
+class Thermostat_TZE200_c88teujp_TemperatureOffset(LocalDataCluster, AnalogOutput):
```

Comment:
Remove manufacturer name from class name:
```python
class TemperatureOffset(LocalDataCluster, AnalogOutput):
```

(or maybe `TuyaTemperatureOffset`)

Reply:
I changed the name to TuyaTemperatureOffset